.travis.yml

```diff
@@ -43,13 +43,14 @@ matrix:
         - os: osx
           env: PYTHON_VERSION=2.7 SETUP_CMD='test'
                CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
+               PIP_DEPENDENCIES='jplephem'
 
         # Check for sphinx doc build warnings - we do this first because it
         # runs for a long time. WCSAxes is a dependency for the docs.
         - os: linux
           env: PYTHON_VERSION=2.7 SETUP_CMD='build_sphinx -w'
                CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
-               PIP_DEPENDENCIES='wcsaxes --no-deps'
+               PIP_DEPENDENCIES='wcsaxes --no-deps jplephem'
```

Comment:
Like this, @bsipocz? Thanks for the help, sorry to be learning by experimentation.